api/app/clients/sms/sap.py

```diff
@@ -119,6 +119,19 @@ def send_sms(self, to, content, reference, sender=None):
                 NOTIFICATION_SENDING
             )
             return order.livelink_order_id[0], NOTIFICATION_SENDING
+        except ApiException as e:
+            if e.status == 401:
+                key = 'auth'
+                auth_api = AuthorizationApi(ApiClient(Configuration(
+                    username=self._client_id,
+                    password=self._client_secret,
+                )))
+
+                resp = auth_api.token_authorization_using_post1('client_credentials')
```

Comment:
we need to try to send the sms again here; not just refresh the token

Reply:
Addressed review comment.